Makefile

```diff
@@ -56,28 +62,31 @@ test-all-with-coverage:
 		-race \
 		-coverprofile=coverage.txt \
 		-covermode=atomic \
-		-tags=memdb,goleveldb,cleveldb,boltdb,rocksdb,grocksdb_clean_link,badgerdb,pebbledb \
```

Comment:
I see the `memdb` tag  was not really used anywhere, but are we triggering the memdb tests at all except in the test-all case?

Reply:
There's no `memdb` tag! It never existed